.kokoro/tests/training/sklearn_structured_hp_tuning.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -eo pipefail
+
+
+download_files() {
```

Comment:
This seems to be a function that we can maybe generalize for all tests and remove from here?

Reply:
Fair point. Perhaps we should do it after this PR i merged, and for all the samples. What do you think?